packages/common/src/api/tan-query/usePurchasesCount.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { useAudiusQueryContext } from '~/audius-query'
+import { ID } from '~/models'
+import { Nullable } from '~/utils/typeUtils'
+
+import { Id } from '../utils'
+
+import { QUERY_KEYS } from './queryKeys'
+
+type Config = {
```

Comment:
Going to continue to advocate that we don't define these each time as their own types but instead just `pick` properties from the tan-query config type. But I'm open to having my mind changed. :-)

Reply:
oh yeah def @schottra ^^ this was just not over-engineering early on, but now i think we should define this in ./types and just use as needed

Reply:
good call. I'm going to add a ticket for this just to keep this change focused for now